tools/README.md

```diff
@@ -167,3 +167,64 @@ The output chunked graph metadata will go as follows (assuming the current direc
     "edge_data": {}
 }
 ```
+
+## Partition Config Conversion
+
+`convert_partition_conf.py` is a tool help you convert old partition config to new version.
+
+### Sample usage like
+
+```
+python tools/convert_partition_conf.py --part-config "{config path}"
```

Comment:
The script name can be more explicit too. How about `tools/change_etype_to_canonical_etype.py`?

Reply:
fixed